<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test1</class>
 <widget class="QDialog" name="test1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="q1b1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>辛纳屈</string>
   </property>
  </widget>
  <widget class="QPushButton" name="q1b2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>海顿</string>
   </property>
  </widget>
  <widget class="QPushButton" name="q1b3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>普雷斯利</string>
   </property>
  </widget>
  <widget class="QLabel" name="q1Label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>381</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>防静电死哦防静电搜发件地佛手机覅等哈粉丝发哈手打发哈刷卡机发哈飞机</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
